README.md

```diff
@@ -1,109 +1,84 @@
-  - Using Read-write repositories for Commands
-  - Using Read-only repositories for Queries (no accidental side-effects against thee data)
-- Fluent Validation
-- AutoMapper
-
-### Domain
-
-- **Strongly Typed IDs**
-  - To combat [Primitive Obsession](https://blog.ndepend.com/code-smell-primitive-obsession-and-refactoring-recipes/) (pass `CustomerId` type into methods instead of `int`, or `Guid`)
-  - Entity Framework can automatically convert the `int`, `Guid`, `nvarchar(..)` to strongly typed ID.
-    - e.g. `13` -> `new CustomerId(13)`
-  - Can be configured per entity!
-
-### Infrastructure
-
-- Entity Framework Core
-  - **Transient Fault Handling**
-- Migrations
-- Data Seeding
-- Model Configuration
-
-### Testing
-
-- Simpler Unit Tests for Application
-  - **No Entity Framework mocking required** thanks to **Specifications**
-- Better Integration Tests
-  - Using [Respawn](https://github.com/jbogard/Respawn) and [TestContainers](https://dotnet.testcontainers.org/)
-  - Integration Tests at Unit Test speed
-  - Test Commands and Queries against a Real database
-  - No Entity Framework mocking required
-  - No need for In-memory database provider
-  - Using [Wire-Mock](https://wiremock.org/) to mock out external services for controlled Integration Tests
-    - e.g. grab real request and responses from external system and then replaying them in the tests
-- Architecture Tests
-  - Using [NetArchTest](https://github.com/BenMorris/NetArchTest)
-  - Know that the team is following the same Clean Architecture fundamentals
-  - The tests are automated so discovering the defects is fast
-- Mutation Testing
-  - Test our tests!
-  - Helps discover the false-positives in our tests
-    - you will know when your tests pass when they should have failed
-  - Inserts bugs into the production code to make sure our tests are effective and testing the right behavior
-  - Using [Stryker Mutator](https://stryker-mutator.io/)
-
-## Publishing Template
+    - e.g. XUnit and NSubstitute packages for all test projects
+- 🧪 Testing
+  - Simpler Unit Tests for Application
+    - **No Entity Framework mocking required** thanks to **Specifications**
+  - Better Integration Tests
+    - Using [Respawn](https://github.com/jbogard/Respawn) and [TestContainers](https://dotnet.testcontainers.org/)
+    - Integration Tests at Unit Test speed
+    - Test Commands and Queries against a Real database
+    - No Entity Framework mocking required
+    - No need for In-memory database provider
+    - Using [Wire-Mock](https://wiremock.org/) to mock out external services for controlled Integration Tests
+      - e.g. grab real request and responses from external system and then replaying them in the tests
+  - Architecture Tests
+    - Using [NetArchTest](https://github.com/BenMorris/NetArchTest)
+    - Know that the team is following the same Clean Architecture fundamentals
+    - The tests are automated so discovering the defects is fast
+  - Mutation Testing
```

Comment:
Are we actually doing Mutation Testing?  If not can we remove this until we've added it in?